includes/html/modal/alert_rule_collection.inc.php

```diff
@@ -83,9 +83,12 @@
                                     dataType: "json",
                                     success: function (data) {
                                         if (data.status == 'ok') {
+                                            $("#search_rule_modal").on('hidden.bs.modal', function(event) {
```

Comment:
Can't you use `one()` instead? No need to have the `off()` then

Reply:
Yes!